packages/devtools/lib/src/logging/logging.dart

```diff
@@ -103,11 +103,13 @@ class LoggingScreen extends Screen {
       // configure the table / details splitter. Setting up this splitter works
       // better once the UI is active in the DOM so we have to delay it until
       // we get the entering event.
-      split.fixedSplitBidirectional(
+      // TODO(devoncarew): Use fixedSplitBidirectional when we move to
+      // package:split v0.0.4.
+      split.flexSplit(
```

Comment:
flex split definitely won't work for the structured logging cases due to flex edge cases. Can you file a bug to track it.

Reply:
Ah, sure, will do.

Reply:
https://github.com/flutter/devtools/issues/914